code_samples/front/render_page/templates/themes/my_theme/layouts/sidebar.html.twig

```diff
@@ -0,0 +1,32 @@
+<div>
+    <div data-ez-zone-id="{{ zones[0].id }}">
+        {% if zones[0].blocks %}
+            {% for block in zones[0].blocks %}
+                <div class="landing-page__block block_{{ block.type }}" data-ez-block-id="{{ block.id }}">
+                    {{ render_esi(controller('EzSystems\\EzPlatformPageFieldTypeBundle\\Controller\\BlockController::renderAction', {
+                        'contentId': contentInfo.id,
+                        'blockId': block.id,
+                        'versionNo': versionInfo.versionNo,
+                        'languageCode': field.languageCode
+                    }))
+                    }}
+                </div>
+            {% endfor %}
+        {% endif %}
+    </div>
+    <div data-ez-zone-id="{{ zones[1].id }}">
+        {% if zones[1].blocks %}
+            {% for block in zones[1].blocks %}
+                <div class="landing-page__block block_{{ block.type }}" data-ez-block-id="{{ block.id }}">
+                    {{ render_esi(controller('EzSystems\\EzPlatformPageFieldTypeBundle\\Controller\\BlockController::renderAction', {
+                        'contentId': contentInfo.id,
+                        'blockId': block.id,
+                        'versionNo': versionInfo.versionNo,
+                        'languageCode': field.languageCode
+                    }))
+                    }}
+                </div>
+            {% endfor %}
+        {% endif %}
+    </div>
+</div>
```

Comment:
Missing newline will make `git` unhappy when adding new lines at the end :smile: